ui/app/templates/exec.hbs

```diff
@@ -29,20 +29,30 @@
   </div>
 </nav>
 
-<div class="tree-and-terminal">
-  <div class="task-group-tree">
-    <h4 class="title is-6">Tasks</h4>
-    <ul>
-      {{#each sortedTaskGroups as |taskGroup|}}
-        <li data-test-task-group>
-          {{exec/task-group-parent
-            taskGroup=taskGroup
-            openInNewWindow=socketOpen
-            activeTaskName=taskName
-            activeTaskGroupName=taskGroupName}}
-        </li>
-      {{/each}}
-    </ul>
+{{#if (eq model.status "dead")}}
+  <div class="tree-and-terminal" data-test-exec-job-dead>
+    <div class="task-group-tree">
+    </div>
+    <div class="terminal-container" data-test-exec-job-dead-message>
+      Job <code>{{model.name}}</code> is dead and cannot host an exec session.
+    </div>
+  </div>
```

Comment:
We can probably use space better with this empty/dead state, but any communication is better than a generic socket error message.

Reply:
ya agreed, I wish I’d thought of this during the design phase 😞